site/docs/components/avatar.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Avatar
+navTitle: Avatar (not built)
+summaryParagraph: An avatar represents an individual or group visually.
+tags: ["Profile picture", "Display picture"]
+needToKnow:
+- Use an avatar to help a user in the platform efficiently identify another person or an organization.
+- Use a profile avatar to represent an individual and an account avatar for an organization.
+- When an avatar photo is unavailable, fall back to initials or first initial.
+
+---
+
+import DosAndDonts from "docs-components/DosAndDonts"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D17270%253A114" allowfullscreen></iframe>
+
+## To keep in mind
+
+- Where possible, link the avatar to details about the individual or group it represents:
+  - Where appropriate, link the avatar to a context-specific view of that individual or group. Otherwise, link to their canonical "profile" page.
+  - Don't link the avatar while it's shown on the individual's canonical "profile" page.
+- When there is no image to show, use initials:
+  - For a single name, use the first letter only, capitalized. For 2 names, use the initial letter of both names, capitalized. For 3 or more names, use the initial letter of the first and last name, capitalized.
+  - When the name is known, use "personal" initials. Otherwise, use "generic" initials.
+- Use with name:
+  - Provide a text label showing the individual or group's name near the avatar when it's the only place the name is shown.
+  - Use an empty `alt=""`.
+- Use without name:
+  - You might skip showing the individual or group's name when the avatar is shown in a list and linked to a place where the name is shown.
+  - For `alt` text, use their name.
+- Loading:
+  - If an image fails to load, fall back to initials.
+
+## When to use and when not to use
+
+<DosAndDonts>
+<Do>
+
+- Use an avatar to help people efficiently identify another person in the platform visually.
+- Use an avatar to distinguish people with the same name.
+
+</Do>
+<Dont>
+
+- Avoid an avatar when most people or groups won't have an image to show.
+
+</Dont>
+</DosAndDonts>
+
+## See also
+
+- [Link vs button](/guidelines/link-vs-button/).
```

Comment:
/meganit Are these two guidelines related?

Reply:
If an avatar is linked to a user profile page, there would be temptation to implement it as a button because it looks more buttony styled than link-ish.